tests/legacy/test_data/monsieurcuisine.testjson

```diff
@@ -0,0 +1,891 @@
+                                    {
+                                        "order": 0,
+                                        "speed": 1,
+                                        "temperature": 100,
+                                        "time": 600,
+                                        "weight": null
+                                    }
+                                ]
+                            },
+                            "ingredients": []
+                        },
+                        {
+                            "id": 12177515,
+                            "translationId": 888319,
+                            "step": 28,
+                            "order": 27,
+                            "name": "250 g Erbsen (TK) und die abgetropften Nudeln in die Auflaufform geben.",
+                            "description": "250 g Erbsen (TK) und die abgetropften Nudeln in die Auflaufform geben.",
+                            "video": null,
+                            "deviceSetting": null,
+                            "ingredients": []
+                        },
+                        {
+                            "id": 12177516,
+                            "translationId": 888320,
+                            "step": 29,
+                            "order": 28,
+                            "name": "Die Hackmischung dazugeben und mit dem Spatel vermengen.",
+                            "description": "Die Hackmischung dazugeben und mit dem Spatel vermengen.",
+                            "video": null,
+                            "deviceSetting": null,
+                            "ingredients": []
+                        },
+                        {
+                            "id": 12177517,
+                            "translationId": 888321,
+                            "step": 30,
+                            "order": 29,
+                            "name": "Den Auflauf mit dem zerkleinerten Emmentaler bestreuen und im vorgeheizten Backofen 30",
+                            "description": "Den Auflauf mit dem zerkleinerten Emmentaler bestreuen und im vorgeheizten Backofen 30 Min. backen.",
+                            "video": null,
+                            "deviceSetting": null,
+                            "ingredients": []
+                        },
+                        {
+                            "id": 12177518,
+                            "translationId": 888322,
+                            "step": 31,
+                            "order": 30,
+                            "name": "Portionsweise auf Teller verteilen und servieren.",
+                            "description": "Portionsweise auf Teller verteilen und servieren.",
+                            "video": null,
+                            "deviceSetting": null,
+                            "ingredients": []
+                        }
+                    ],
+                    "ingredientGroups": [
```

Comment:
This library supports grouping of ingredients (see the documentation at https://github.com/hhursev/recipe-scrapers/blob/4da3a6493cebe91c85378a2307645006b4fb17a9/docs/in-depth-guide-ingredient-groups.md) - however, please consider that optional to implement.  We can add it in future; getting the scraper working again doesn't need to wait for that.

Reply:
Yeah I won't do that now